File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/importering/SykefraværsstatistikkNæring.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.importering;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
public class SykefraværsstatistikkNæring implements Sykefraværsstatistikk {
    private int årstall;
    private int kvartal;
    private String næringkode;
    private int antallPersoner;

    private BigDecimal tapteDagsverk;
    private BigDecimal muligeDagsverk;
}

File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/domenemodeller/SykefraværsstatistikkNæring.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domenemodeller

import java.math.BigDecimal

data class SykefraværsstatistikkNæring(
    override val årstall: Int,
    override val kvartal: Int,
    val næringkode: String,
    override val antallPersoner: Int,
    override val tapteDagsverk: BigDecimal?, // Må være nullable fordi klassen brukes som en kafkamelding ...
    override val muligeDagsverk: BigDecimal?, // Må være nullable fordi klassen brukes som en kafkamelding ...
) : Sykefraværsstatistikk